src/support/ManagedGroupSummary.ts

```diff
@@ -0,0 +1,90 @@
+import { ButtonPrimary } from '@terra-ui-packages/components';
+import { Fragment, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { TextArea, TextInput } from 'src/components/input';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { useOnMount } from 'src/libs/react-utils';
+import { ResourceTypeSummaryProps } from 'src/support/SupportResourceType';
+
+export const ManagedGroupSummary = (props: ResourceTypeSummaryProps) => {
+  const { query } = Nav.useRoute();
+  const [groupName, setGroupName] = useState(props.fqResourceId.resourceId || '');
+  const [groupSummaryInfo, setGroupSummaryInfo] = useState('');
+  const [groupPolicies, setGroupPolicies] = useState('');
+
+  const submit = async (): Promise<void> => {
+    setGroupSummaryInfo('');
+    setGroupPolicies('');
+    const groupSummaryInfo = await Ajax().Groups.group(groupName).getSupportSummary();
+    const groupPolicies = await Ajax().SamResources.getResourcePolicies(props.fqResourceId);
+    setGroupSummaryInfo(JSON.stringify(groupSummaryInfo, null, 2));
+    setGroupPolicies(JSON.stringify(groupPolicies, null, 2));
+  };
```

Comment:
It might be nice to have some kind of busy state / spinner here in case these requests take some time.

Reply:
Wrapping with error handling would also be good. I'm getting a 403, but there's no indication of anything happening when I press Submit.

Reply:
I added error handling. I didn't quite follow how the spinners worked in other places so deemed it not critical in this instance :)